site/source/docs/compiling/Building-Projects.rst

```diff
@@ -73,6 +73,28 @@ WebAssembly.
   not the native system compiler. If *emcc* is not used, you may need to modify
   the configure or cmake scripts.
 
+.. _building-projects-with-cmake:
+
+Using CMake with Emscripten
+==================================
+
+If you have not already, first, you have to install and activate the SDK tag you wish to use, and after, you must add the environment variables in your terminal.
+
+.. code-block:: bash
+
+  ./emsdk install latest  # Download and install the latest SDK. You probably already did this!
+  ./emsdk activate latest # Make the "latest" SDK "active" for the current user
+  source ./emsdk_env.sh   # Activate PATH and other environment variables in the existing terminal
```

Comment:
this duplicates things from the download page. I think it would be better to say "you need to install and activate the sdk, and then you will have emcmake in the path", or something similar, with a link to the download page.

Reply:
the important bit is actually the `source ./emsdk_env.sh` but I am having trouble figuring how to naturally tell people about using it. Without it, there will be no `emcmake` available to use.

Reply:
Ah, also, I don't think the Download page is a good place to put the install and activate steps, downloading is something you do once. 

These are steps people will do more often.

Reply:
I think its OK to assume that all the tools are in the path at this point in the instructions.  You can maybe say something like `Make sure you have already install emcripten so that emcc and other tools are in your PATH already.`    But I don't think we need to repeat the emsdk steps here.

Reply:
I will mention that but a least add the example between paatentesis of source usage. Writing in cryptic wording is less helpful than examples.